tests/acceptance/features/apiOcm/share.feature

```diff
@@ -1229,3 +1229,21 @@ Feature: an user shares resources using ScienceMesh application
         }
       }
       """
+
+  @issue-10689
+  Scenario: federation user lists all the spaces
+    Given using server "REMOTE"
+    And "Brian" has created the federation share invitation
+    And using server "LOCAL"
+    And "Alice" has accepted invitation
+    And user "Alice" has created folder "folderToShare"
+    And user "Alice" has sent the following resource share invitation to federated user:
+      | resource        | folderToShare |
+      | space           | Personal      |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | Editor        |
+    And using server "REMOTE"
+    When user "Brian" lists all available spaces via the Graph API
+    Then the HTTP status code should be "200"
+    And the json responded should not contain a space with name "folderToShare"
```

Comment:
```suggestion
    And the json response should not contain a space with name "folderToShare"
```
Looks like a typo. If so we need to fix it on another PR.

Reply:
yeah we can do that.

Reply:
Please, do the follow up immediately once this PR is merged